test/testing-api.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path')
+const os = require('os')
+
+const startSSB = require('../ssb')
+const startAPI = require('../api')
+
+module.exports = function testingAPI (opts = {}) {
+  if (!opts.name) {
+    opts.name = `buttcast-test-${Date.now()}-${Math.floor(Math.random() * 1000)}`
+  }
+  if (!opts.path) opts.path = path.join(os.tmpdir(), opts.name)
```

Comment:
this is our test server!
The main difference here is that it sets up a db (database) in your computers "temp" folders. This means your dev environment won't get messed with every time you run the tests. They temp folders get cleared each time you restart